tt_metal/hw/inc/ethernet/dataflow_api.h

```diff
@@ -38,20 +40,14 @@ volatile uint32_t *RtosTable =
 
 void (*rtos_context_switch_ptr)();
 
+namespace internal_ {
```

Comment:
@tt-aho namespacing some functions that should only be called from within ethernet dataflow implementation